### DISCUSSION TITLE
Cannot send email

### DISCUSSION THREAD

**Discussion by make-github-pseudonymous-again** [edemaine/coauthor#202] [Installation] [answered]
I installed postfix, configured it as described in the README and started it. However the server is not able to send emails.

    ====== BEGIN MAIL #1 ======
    (Mail not sent; to enable sending, set the MAIL_URL environment variable.)
    ...

What is this environment variable for and what should it contain?

**Comment by edemaine** [comment, marked as answer]
https://docs.meteor.com/api/email.html
If you're working locally, then you want to set `MAIL_URL` to `smtp://your.host.name:25/`, as in [`.deploy/mup.js`](https://github.com/edemaine/coauthor/blob/master/.deploy/mup.js).  (Are you using `mup`?)

**Comment by make-github-pseudonymous-again** [comment]
Thanks! Not using mup for the moment but might in the future to avoid the hastle.
